setup.py

```diff
@@ -15,6 +15,8 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+# Galaxy's runtime environment and packages are implicitly assumed to be present, and only
+# explicitly installed for tests.
 REQS_FULL = [
```

Comment:
ruamel.yaml and requests seems to be a core dependency
watchdog is a conditional dependency, which we could install if TPV is used.

https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/dependencies/pinned-requirements.txt#L160